selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -239,4 +307,81 @@ private void createFile(Path pathToRootContentDirectory, Path pathToFile) throws
   public String getFileContent(String pathToFile) throws IOException {
     return IOUtils.toString(ghRepo.getFileContent(pathToFile).read(), "UTF-8");
   }
+
+  public String getRepoSha() throws IOException {
```

Comment:
I would rather name it `getDefaultBranchSha()`. 
Alsod `Repo` term is redundant at all so as it is the method of `TestGitHubRepository` class.